source/codegen/metadata/nifgen/functions.py

```diff
@@ -1471,6 +1537,30 @@
         ],
         'returns':'ViStatus'
     },
+    'ErrorHandler':{
```

Comment:
Note: Attaching screenshot of chm for `ErrorHandler` API to help review the metadata:

![chm](https://user-images.githubusercontent.com/77618357/120766281-b413c600-c537-11eb-92fa-efc0f8d22534.PNG)